ivy/functional/frontends/paddle/tensor/math.py

```diff
@@ -352,3 +352,11 @@ def asinh(x, name=None):
 @to_ivy_arrays_and_back
 def max(x, axis=None, keepdim=False, name=None):
     return ivy.max(x, axis=axis, keepdims=keepdim)
+
+
+@with_supported_dtypes(
+    {"2.5.0 and below": ("float16","float32", "float64")}, "paddle"
```

Comment:
not sure if I missed anything, it seems like it can take `int32` and `int64` from the docs? https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/heaviside_en.html

Reply:
@juliagsy  on it.